app/handlers/clap.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const check = require(__base + "/app/modules/common/checkpost");
```

Comment:
check function is not a good name.
Why dont we use a simple validation function.
Create a clap file inside post module and use that file to create validation function, and increase clap function. 
Also checkpost cannot be inside modules/common

Reply:
I thought we need to check post not only for clapping rather we would also need it for number of other things like fetching the post, or while commenting on the post, or sharing post. This thing must exist so created one universal validator under the common. Let me know what you say about it.